static/src/javascripts/projects/common/modules/commercial/ads/user-ad-feedback.js

```diff
@@ -0,0 +1,48 @@
+define([
+    'bonzo',
+    'qwery',
+    'common/utils/fetch',
+    'common/utils/config'
+], function (
+    bonzo,
+    qwery,
+    fetch,
+    config
+) {
+
+    function userAdFeedback(pagePath, adSlotId, creativeid, reason) {
+        // TODO: This. Better.
+        if (reason !== 'ad-feedback-menu-opened') {
+            return recordUserAdFeedback(pagePath, adSlotId, creativeid, reason);
+        } else {
+            return recordUserAdFeedback(pagePath, adSlotId, creativeid, 'ad-feedback-menu-opened');
```

Comment:
But if the code enters this branch, that means `reason === 'ad-feedback-menu-opened'` and so
```
return recordUserAdFeedback(pagePath, adSlotId, creativeid, 'ad-feedback-menu-opened');
```
is equivalent to
```
return recordUserAdFeedback(pagePath, adSlotId, creativeid, reason);
```
and thus you don't need the if at all. Consequently, you don't need `userAdFeedback` either.